### PR TITLE
Add redux examples

### PR DESCRIPTION
**Pull request by ekosz**
1. Add example for connected redux application
2. Add example for universal connected redux application

This is almost done, but I've gotten stuck on an error with the universal version. The server renders the page correctly, but then the front-end errors out with:

```
Uncaught (in promise) TypeError: Object is not async iterable
```

It seems to be a problem with `getStoreRenderArgs`. @taion have you seen an error like that in the past?

### CODE REVIEW

**Comment by taion** [comment]
yes... sigh. it's a core-js bug, https://github.com/zloirock/core-js/issues/262 / https://github.com/zloirock/core-js/pull/267.

see babel issue: https://github.com/babel/babel/issues/4783
and discussion of workaround used elsewhere: https://github.com/ctrlplusb/react-universally/pull/182

i don't have a good answer for this right now

**Comment by taion** [comment]
(i mean, the pr i have up on core-js, sorta, but i'm not sure it's correct)

**Comment by ekosz** [comment]
Ah, cool. Not a problem. When you get a chance can you look over the examples and make sure they're idiomatic for `found`? Other than than error it should all be working.

**Comment by taion** [comment]
Sorry – I've been a little swamped this week. I haven't forgotten about this PR, I just haven't had a chance to look more closely yet.

**Comment by ekosz** [comment]
@taion Not a problem at all. I know you're busy, don't worry about.

**Comment by taion** [comment]
https://github.com/babel/babel/pull/5195 would also fix the problem

File: examples/universal-connected-redux/src/server.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <title>Found Universal Redux Example</title>
+</head>
+
+<body>
+  <div id="root">${ReactDOMServer.renderToString(element)}</div>
+  <script>
+    window.__PRELOADED_STATE__ = ${JSON.stringify(state)};
```
Comment (taion) [inline examples/universal-connected-redux/src/server.js]: xss vulnerability here

File: examples/universal-connected-redux/src/routeConfig.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        ),
+      },
+      new Redirect({
+        from: 'baz',
+        to: '/foo',
+      }),
+    ],
+  },
+];
+
```
Comment (taion) [inline examples/universal-connected-redux/src/routeConfig.js]: extraneous newline?

File: examples/universal-connected-redux/src/App.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+      {children}
+    </div>
+  );
+}
+
+App.propTypes = propTypes;
+
+export default App;
+
```
Comment (taion) [inline examples/universal-connected-redux/src/App.js]: extraneous newline?

File: examples/connected-redux/src/index.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```
Comment (taion) [inline examples/connected-redux/src/index.js]: do you think it'd make sense to put everything here, for this example specifically? it's how i have it in the others... i think it'd still be short enough to be readable

File: examples/connected-redux/package.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "connected-redux",
```
Comment (taion) [inline examples/connected-redux/package.json]: i would just call this "redux"

File: examples/universal-connected-redux/package.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "universal-connected-redux",
```
Comment (taion) [inline examples/universal-connected-redux/package.json]: i would just call this "universal redux"

**Comment by taion** [comment]
@ekosz BTW the Babel issue got fixed in babel 6.23.

**Comment by ekosz** [comment]
@taion Awesome. I'll update these examples after work today

**Comment by taion** [comment]
Well, as soon as babel-runtime 6.23 gets published, anyway.

File: examples/connected-redux/src/index.js (hunk 4 above)
Comment (ekosz) [inline examples/connected-redux/src/index.js]: Yeah not a problem.

**Comment by ekosz** [comment]
@taion Updated the examples. Unfortunately even after updates all of the dependencies I'm still getting that error. It looks like `babel-plugin-transform-runtime@6.23.0` was released, but other modules are still relying on the old plugin?

**Comment by taion** [comment]
babel-runtime 6.23.x actually hasn't been released yet... that's the package that needs the update, sorry

**Comment by taion** [comment]
This looks great though. I'll merge once Babel updates.

**Comment by taion** [comment]
Oops, got sidetracked. Thanks again!

**Comment by ekosz** [comment]
@taion Not a problem. Glad I could give back in a small way.

**Pull request merged**